shared/src/business/entities/trialSessions/NewTrialSession.js

```diff
@@ -27,7 +27,7 @@ NewTrialSession.prototype.init = function init(
 NewTrialSession.VALIDATION_ERROR_MESSAGES = {
   ...TrialSession.VALIDATION_ERROR_MESSAGES,
   alternateTrialClerkName:
-    'A valid Alternate Trial Clerk name must be provided if "Other*" is selected',
+    'A valid Alternate Trial Clerk name must be provided if "Other" is selected',
```

Comment:
This probably doesn't matter, but should the validation message's "Alternate Trial Clerk" be lowercase like in `SessionAssignmentsForm`?

Reply:
Good catch, thank you! That would be better